src/firebase/addSettlement.js

```diff
@@ -6,4 +6,6 @@ export const addSettlement = async (entry) => {
     groupID,
     settleDate,
   });
+
+  console.log("document successfully written");
```

Comment:
Das wird auch geloggt wenn die Query failed. 
Entweder müssen wir den Datenbank-Call in ein Try/Catch packen oder weglassen (YOLO)